simple_history/manager.py

```diff
@@ -118,16 +114,13 @@ def __init__(self, model, instance=None):
         self.model = model
         self.instance = instance
 
-    def get_super_queryset(self):
-        return super().get_queryset()
-
     def get_queryset(self):
-        qs = self.get_super_queryset()
+        qs = super().get_queryset()
         if self.instance is None:
             return qs
 
-        key_name = get_app_model_primary_key_name(self.instance)
-        return self.get_super_queryset().filter(**{key_name: self.instance.pk})
+        pk_name = utils.get_pk_name(self.instance._meta.model)
```

Comment:
```suggestion
        pk_name = utils.get_pk_name(self.model)
```
Does that work just as well?